datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/marketplace/README.md

```diff
@@ -20,5 +20,8 @@ Specific step-by-step configuration instructions for this integration.
 
 Screenshots are okay for this section, if they help demonstrate configuration steps.
 
+## Uninstallation
```

Comment:
```suggestion
## Uninstall
```

Reply:
Hi Heston, the product spec mentioned the header being "Uninstallation." Pinging @eho1307

Reply:
@hestonhoffman I was following the same format as 'configuration'. Let me know if docs has a strong preference for 'uninstall'

Reply:
The reason I suggested "Uninstall" is because "Uninstallation" is not a word. This is merely a suggestion, though. There's a fair amount of inconsistency in these templates already. If you do want to go for consistency, I'd change a lot of these words to gerund forms like "Setting up", "Uninstalling", "Configuring" etc.

Let me know what you decide on here before you merge the PR in though, cos I need to adjust our build scripts accordingly to make sure the section is removed from the output.

Reply:
@hestonhoffman @alai97 The team discussed this and decided to go with 'Uninstallation' because Marketplace tiles currently say 'Configuration' and we'd like them to match. If Docs has suggestions on consistency for the overall tile, we can make those changes when we update the overall Marketplace docs. Does that work?

Reply:
Cool, I've changed the build scripts to use "uninstallation," so you're good to merge this

Reply:
@hestonhoffman. Rerequested your review because added punctuation to be consistent across the README template.